docs/zig-package-manager.org

```diff
@@ -0,0 +1,54 @@
+** Zig package manager
+Since 0.11, Zig has now an official built-in package manager named.
+
+There is no global repository like crates.io(Rust) or npmjs.com(Node.js) for Zig packages. Instead, Zig packages are distributed as git repositories.
+
+In order to add packages to your project you are going to create and edit a file in the root of your project named "build.zig.zon" (zon stands for Zig Object Notation, which is an equivalent to json).
+
+*** Add a package in your project
+1. Add the packages you want in your build.zig.zon file
+#+begin_src zon
+.{
+    .name = "name-of-my-project",
+    .version = "0.0.1",
+
+    .dependencies = .{
+        // Here you add all the packages you want, in this exemple I have imported zap which is a web framework
+        .zap = .{
+            .url = "https://github.com/zigzap/zap/archive/refs/tags/v0.1.7-pre.tar.gz",
+            .hash = "1220002d24d73672fe8b1e39717c0671598acc8ec27b8af2e1caf623a4fd0ce0d1bd",
+        },
+    }
+}
+#+end_src
+You can get further understanding in how to write your build.zig.zon following the official doc: https://github.com/ziglang/zig/blob/master/doc/build.zig.zon.md
+
+TODO: comment trouver le hash ? zig fetch avec 0.12 ou ?
+Jai des erreurs Tls
+
+2. Add those added packages in your build.zig
+Simply add the following code after the "addExectuable" function
+#+begin_src zig
+const zap = b.dependency("zap", .{
+        .target = target,
+        .optimize = optimize,
+});
+exe.addModule("zap", zap.module("zap"));
+exe.linkLibrary(zap.artifact("facil.io"));
+#+end_src
+Note: here it is specific to the zap package, you might have different options and no need to link a library for an other package (zap uses facil.io under the hood)
+
+3. Simply "@import" them and use them in your code :)
+#+begin_src zig
+const zap = @import("zap");
+#+end_src
+
+
+*** Sources
```

Comment:
Maybe insert the sources into the bibliorgraphy file and add citation (create a cool bibliography when exporting it into html)

Reply:
yes I havent had time to look into zotero and others yet, but thats something im gonna do soon.
since I have to submit something for tonight I am not losing time doing this yet